src/axom/sidre/interface/c_fortran/wrapView.h

```diff
@@ -34,164 +37,186 @@ extern "C" {
 // splicer begin class.View.C_declarations
 // splicer end class.View.C_declarations
 
-SIDRE_IndexType SIDRE_View_get_index(SIDRE_View* self);
+SIDRE_IndexType SIDRE_View_get_index(SIDRE_View *self);
 
-const char* SIDRE_View_get_name(const SIDRE_View* self);
+const char *SIDRE_View_get_name(const SIDRE_View *self);
 
-void SIDRE_View_get_name_bufferify(const SIDRE_View* self,
-                                   char* SHF_rv,
-                                   int NSHF_rv);
+void SIDRE_View_get_name_bufferify(const SIDRE_View *self,
+                                   char *SHC_rv,
+                                   int SHT_rv_len);
 
-void SIDRE_View_get_path_bufferify(const SIDRE_View* self,
-                                   char* SHF_rv,
-                                   int NSHF_rv);
+void SIDRE_View_get_path_bufferify(const SIDRE_View *self,
+                                   char *SHC_rv,
+                                   int SHT_rv_len);
 
-void SIDRE_View_get_path_name_bufferify(const SIDRE_View* self,
-                                        char* SHF_rv,
-                                        int NSHF_rv);
+void SIDRE_View_get_path_name_bufferify(const SIDRE_View *self,
+                                        char *SHC_rv,
+                                        int SHT_rv_len);
 
-SIDRE_Group* SIDRE_View_get_owning_group(SIDRE_View* self, SIDRE_Group* SHC_rv);
+SIDRE_Group *SIDRE_View_get_owning_group(SIDRE_View *self, SIDRE_Group *SHC_rv);
 
-bool SIDRE_View_has_buffer(const SIDRE_View* self);
+bool SIDRE_View_has_buffer(const SIDRE_View *self);
```

Comment:
Gut check: There are lots of small formatting changes.
Did you run `make style` after `make generate`?

Reply:
I ran 'make style' again to check, and there were no new changes. I ran with `VERBOSE=1` to make sure that the generated files were being formatted, and they were. It's using executable at `/collab/usr/gapps/axom/devtools/toss_4_x86_64_ib/latest/llvm-10.0.0/bin/clang-format`, which seems correct.

If I edit the file to make it `Sidre_View*`, clang-format changes it to `Sidre_View *`, yet running `clang-format` on the `develop` branch preserves it as `Sidre_View*`.

Reply:
Thanks for checking!